frontend/src/scenes/data-warehouse/editor/OutputPane.tsx

```diff
@@ -89,7 +89,12 @@ export function OutputPane(): JSX.Element {
         return response?.results?.map((row: any[]) => {
             const rowObject: Record<string, any> = {}
             response.columns.forEach((column: string, i: number) => {
-                rowObject[column] = row[i]
+                // Handling objects here as other viz methods can accept objects. Data grid does not for now
+                if (typeof row[i] === 'object' && row[i] !== null) {
+                    rowObject[column] = JSON.stringify(row[i])
+                } else {
+                    rowObject[column] = row[i]
+                }
```

Comment:
Probably worth revisiting some kind of object handling here, but it seems like the customer is throwing, so let's get this out.